src/interfaces/Periodic-props.ts

```diff
@@ -0,0 +1,28 @@
+export interface PeriodicProps {
```

Comment:
Name of file should be in lower case and hypen delimited. You can use `element-props.ts`.